### PR TITLE
feat: correct types for logo

### PR DESCRIPTION
**Pull request by danielgent**
This _should_ fix the email logo sending issue :smile:

### CODE REVIEW

File: src/types/Client.ts [hunk 1]
```diff
@@ -54,7 +54,6 @@ export const BaseClientSchema = z.object({
   name: z.string(),
   audience: z.string(),
   language: z.string().length(2).optional(),
-  logo: z.string().optional(),
```
Comment (danielgent) [inline src/types/Client.ts:57]: this matches what we have in the KV store

File: src/controllers/email.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export async function sendEmailValidation(
   const language = client.language || "en";
 
   const logo = getClientLogoPngGreyBg(
-    client.logo ||
```
Comment (markusahlstrand) [inline src/controllers/email.ts:31]: It's odd that this compiled before? Guess this doesn't exist on the client type?

File: src/types/Client.ts (hunk 1 above)
Comment (markusahlstrand) [reply src/types/Client.ts:57]: Ahh, that's why it compiled...

**Review by markusahlstrand — APPROVED** [review]
Looks good!

File: src/types/Client.ts (hunk 1 above)
Comment (danielgent) [reply src/types/Client.ts:57]: Yes I think it was a case of this change should have been in this PR https://github.com/sesamyab/auth/pull/93

**Pull request merged**